packages/shared/src/components/onboarding/OnboardingFooter.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { wrapperMaxWidth } from './common';
+import SignupDisclaimer from '../auth/SignupDisclaimer';
+import TrustedCompanies from '../TrustedCompanies';
+import { IconSize } from '../Icon';
+
+export const OnboardingFooter = (): ReactElement => (
+  <footer
+    className={classNames(
+      'flex h-full max-h-[10rem] w-full px-4 tablet:px-6',
+      wrapperMaxWidth,
+    )}
+  >
+    <div className="relative flex flex-1 flex-col gap-6 pb-6 tablet:mt-auto laptop:mr-8 laptop:max-w-[27.5rem]">
+      <SignupDisclaimer className="mb-0 hidden tablet:mb-10 tablet:block" />
+
+      <TrustedCompanies
+        iconSize={IconSize.Small}
+        reverse
+        className=" mt-5 block tablet:hidden"
+      />
```

Comment:
I remember seeing some differences in these things showing or not in mobile or at all (trusted companies for ex here is only mobile, pretty sure at some point I've seen designs without it). Just since u're at it, might be worth just pinging design/product if we want them or we can just remove 😈

Reply:
Ah good idea indeed! I'll check with them :)

Reply:
https://dailydotdev.slack.com/archives/C02AA18RD1D/p1729075784057059